erigon-lib/metrics/register.go

```diff
@@ -117,13 +115,13 @@ func GetOrCreateGauge(name string) Gauge {
 //   - foo, with labels []string{"bar", "baz"}
 //
 // The returned GaugeVec is safe to use from concurrent goroutines.
-func GetOrCreateGaugeVec(name string, labels []string, help ...string) *prometheus.GaugeVec {
+func GetOrCreateGaugeVec(name string, labels []string, help ...string) *GaugeVec {
```

Comment:
I see some native funcs: 
`getOrCreateMetricWithLabels` it accepting type `labels Labels` where 
and it's `type Labels map[string]string`

Reply:
Can you elaborate a bit?